src/components/ContentWrapper.jsx

```diff
@@ -367,12 +369,21 @@ const ContentWrapper = (props) => {
       || seuratRunning || seuratRunningError
     );
 
+    const {
+      DATA_EXPLORATION, DATA_MANAGEMENT, DATA_PROCESSING, PLOTS_AND_TABLES,
+    } = modules;
+
+    // disable links if user is not in one of the experiment analysis modules
+    const nonExperimentModule = ![DATA_EXPLORATION,
+      DATA_MANAGEMENT, DATA_PROCESSING, PLOTS_AND_TABLES]
+      .includes(currentModule) && disableIfNoExperiment;
```

Comment:
what is, and is not, an experiment analysis module?

Reply:
Based on the odd ones out of `modules`, I would say they are `REPOSITORY` and `SETTINGS`, but @StefanBabukov might know more

Reply:
ahh okay gotcha makes sense -- this is disabling the links if in the datasets repository or settings page